index.js

```diff
@@ -204,9 +213,7 @@ class Link {
       cb = () => {}
     }
 
-    const opts = _.defaults({}, _opts, {
-      retry: 3
-    })
+    const opts = { retry: 3, ..._opts }
```

Comment:
the change of order will change the result, is this intentional?

Reply:
actually this is correct based on docs https://lodash.com/docs/4.17.15#defaults, left hand side params are considered final
```js
_.defaults({ 'a': 1 }, { 'b': 2 }, { 'a': 3 });
// => { 'a': 1, 'b': 2 }
```